fiaas_skipper/deploy/bootstrap.py

```diff
@@ -0,0 +1,31 @@
+import logging
+
+import pinject
+from k8s.models.common import ObjectMeta
+from k8s.models.job import JobSpec, Job
+from k8s.models.pod import Container, PodSpec, PodTemplateSpec
+
+LOG = logging.getLogger(__name__)
+
+
+class FiaasBootstrap(object):
+    @pinject.copy_args_to_internal_fields
+    def __init__(self, deployment_config, channel):
+        pass
+
+    def __call__(self):
```

Comment:
Is it better to have a class that we need to instantiate than an static function that receives `deployment_config` as parameter?

Reply:
You are right it's probably not. I'll try this with just making it a static function instead.